### ISSUE TITLE
Won't work because mplayer/ffmpeg can't handle https

### ISSUE THREAD

**Issue by diimdeep** [uschek/somafm#3]
Running `somafm play groovesalad` don't play anything.   
Gives error when press `m`

```
  Playing   SomaFM Groove Salad

events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: This socket has been ended by the other party
    at Socket.writeAfterFIN [as write] (net.js:285:12)
    at ReadStream.<anonymous> (/Users/wolf/.nvm/versions/node/v4.8.0/lib/node_modules/somafm/cli.js:149:25)
    at emitOne (events.js:77:13)
    at ReadStream.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at ReadStream.Readable.push (_stream_readable.js:111:10)
    at TTY.onread (net.js:537:20)
```

Problem is probably with https links

https://api.somafm.com/channels.json returns https links to playlists

running `mplayer -playlist https://api.somafm.com/groovesalad256.pls` returns

`https protocol not found, recompile FFmpeg with openssl, gnutls or securetransport enabled.`

running `mplayer -playlist http://api.somafm.com/groovesalad256.pls` works

Tried reinstall ffmpeg and mplayer.
upd: looks like in my case mplayer uses internal ffmpeg

```
$nvm current
v4.8.0

$brew reinstall ffmpeg --with-libvpx --with-libvorbis --with-openssl --with-theora --with-x265 --with-fdk-aac
$brew install mplayer --HEAD
```

**Comment by uschek** [comment]
The playlist files in `channels.json` are always referenced via HTTPS:

- [channels.json (HTTPS)](https://api.somafm.com/channels.json)
- [channels.json (HTTP)](http://api.somafm.com/channels.json)

Fetching the channels per HTTP won't fix your problem. I'm thinking about parsing the playlist files manually and passing the direct stream URL (`http://ice1.somafm.com/groovesalad-256-mp3`) to MPlayer. That should solve it.

**Comment by uschek** [comment]
Just released [2.0.0](https://github.com/uschek/somafm/releases/tag/v2.0.0). Direct stream URLs are passed to MPlayer now instead of playlist URLs.

@diimdeep The HTTPS problems should be fixed now. Feel free to reopen this issue if it still doesn't work for you.
